src/libra_py/workflows/nbra/step2.py

```diff
@@ -265,6 +265,9 @@ def run(params):
             # The input file template defined in params["prefix0"] should have   prefix = 'x0' !
             os.system("cp %s/curr0/x0.export/index.xml %s/x0_index_%i.xml" % (wd, rd, t))
             os.system("cp %s/curr0/x0.save/data-file-schema.xml %s/x0_data-file-schema_%i.xml" % (wd, rd, t))
+            os.system("cp %s/curr0/x0.save/charge-density.dat %s/x0_charge-density_%i.dat" % (wd, rd, t))
+            os.system("cp %s/curr0/x0.save/wfcdw1.dat %s/x0_wfcdw1_%i.dat" % (wd, rd, t))
```

Comment:
please add a flag to control which files to write - set the flags to 1 (write all the files) by the defaults. This may be needed if users don't have a lot of storage space to keep all these extrafiles - I imagine those might be heavy